pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,22 +125,81 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterPersistentVolumeClaimChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
In my eyes, we should check if the namespace is matched with an added/updated PVC at least.

Reply:
Thank you for the suggestion. I'll fix this problem.

Reply:
I fixed this problem.